dstk-web/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
```

Comment:
I forget how `.gitignore` precedence works. The top-level `.gitignore` is whitelist-based, so everything is automatically excluded unless otherwise specified. I don't know offhand if this one will replace the top-level rules entirely, or if they become additive/subtractive. Might be worth checking

Reply:
The `.gitignore` in the frontend directory will overwrite any overlapping rules from the top-level one.

I don't think there's any foul play between the two but I'm not opposed to just having a root-level one either. The only additions I would add to the top-level one are the `dist` directory for the compiled TypeScript and possibly the `.DS_Store` file.

Reply:
Oh no this is totally fine, more just a passing "oh that's a random factoid I don't know" moment